e2e/cypress/integration/account_settings/display/account_settings_positiion_spec.js

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @account_setting
+import * as TIMEOUTS from '../../../fixtures/timeouts';
```

Comment:
Please remove `@prod`. We'll add it back once we found the tests to be stable for a few runs.

```suggestion
// Group: @account_setting

import * as TIMEOUTS from '../../../fixtures/timeouts';
```